Makefile

```diff
@@ -76,8 +78,10 @@ test-skopeo:
 		skopeo_path=$${GOPATH}/src/github.com/containers/skopeo && \
 		vendor_path=$${skopeo_path}/vendor/github.com/containers/image && \
 		git clone -b $(SKOPEO_BRANCH) https://github.com/$(SKOPEO_REPO) $${skopeo_path} && \
-		rm -rf $${vendor_path} && cp -r . $${vendor_path} && rm -rf $${vendor_path}/vendor && \
 		cd $${skopeo_path} && \
+		env GO111MODULE="on" \
+		go mod edit -replace github.com/containers/image=$(PROJECT_PATH) && \
```

Comment:
(Absolutely non-blocking nits:
- The `env` is probably not necessary (at least it’s not used in most of the other `GO111MODULE` invocations)
- The single-use `PROJECT_PATH` variable seems not worth the overhead of introducing an extra name, but opinions on that may well differ.

)